srcpkgs/clevis/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'clevis'
+pkgname=clevis
+version=20
+revision=1
+build_style=meson
+hostmakedepends="asciidoc cryptsetup curl jq keyutils pkg-config tpm2-tools"
+makedepends="bash-completion cryptsetup-devel dracut jansson-devel
+ jose-devel libaudit-devel luksmeta-devel udisks2-devel"
+depends="bash cryptsetup curl libpwquality tpm2-tools"
+short_desc="Pluggable framework for automated decryption"
+maintainer="Johannes Heimansberg <git@jhe.dedyn.io>"
+license="GPL-3.0-or-later WITH custom:OpenSSL-Exception"
```

Comment:
openssl exception shouldn't be custom

Reply:
Is there an SPDX identifier for that? xlint complains about it and I couldn't find and identifier on the spdx.org website, which is why I declared it as custom and included the license file.

Reply:
ok i [misremembered](https://github.com/void-linux/void-packages/pull/38344#discussion_r932635161)